tests/test_rewrite_rules/test_rewrite_rules.py

```diff
@@ -32,141 +29,134 @@ class TestEntry:
 
 def build_test_entry_set(input_url: Namespace, model: str) -> List[TestEntry]:
     return [
-        TestEntry(input_url, f"includes/{model}"),
-        TestEntry(input_url, f"includes/{model}.yaml", "text/yaml"),
-        TestEntry(input_url, f"includes/{model}.ttl", "text/turtle"),
-        TestEntry(input_url, f"includes/{model}.jsonld", "application/json"),
-        TestEntry(input_url, f"includes/{model}.shex", "text/shex"),
-        TestEntry(input_url[".context.jsonld"], f"includes/{model}.context.jsonld"),
-        TestEntry(input_url[".owl"], f"includes/{model}.owl"),
-        TestEntry(input_url["/"], f"includes/{model}/"),
+        TestEntry(input_url, f"linkml-model/docs/{model}"),
+        TestEntry(input_url, f"linkml-model/linkml_model/model/schema/{model}.yaml", "text/yaml"),
+        TestEntry(input_url, f"linkml-model/linkml_model/rdf/{model}.ttl", "text/turtle"),
+        TestEntry(input_url, f"linkml-model/linkml_model/json/{model}.json", "application/json"),
+        TestEntry(input_url, f"linkml-model/linkml_model/shex/{model}.shex", "text/shex"),
+        TestEntry(input_url[".context.jsonld"], f"linkml-model/linkml_model/jsonld/{model}.context.jsonld"),
+        TestEntry(input_url[".owl"], f"linkml-model/linkml_model/owl/{model}.owl.ttl"),
+        TestEntry(input_url["/"], f"linkml-model/{model}/"),
     ]
 
 
-class TestLists:
-    __test__ = False
+def generate_fixture_lists():
+    server = os.environ.get("SERVER", DEFAULT_SERVER)
```

Comment:
implicit env var: when would this need to change?

Reply:
This was previously done in the unittest main clause, which I removed for pyunit style.
[httpd/README.md Testing the Rewrite Rules](https://github.com/linkml/linkml/tree/main/tests/test_rewrite_rules/httpd#testing-the-rewrite-rules) addresses this question, 
I can add a clearer link to it in the code
Also noticed the test command would need to be updated in those instructions.